packages/components/src/RadarChart/RadarChart.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Radar, RadarChart, PolarAngleAxis, Dot, LabelWithClick } from './RadarChart.component';
+
+describe('RadarChart', () => {
+	it('should render a RadarChart', () => {
+		const props = {
+			data: [
+				{ axis: 'Validity', A: 4 },
+				{ axis: 'Social curation', A: 3 },
+				{ axis: 'Completeness', A: 2 },
+				{ axis: 'Discoverability', A: 1 },
+				{ axis: 'other', A: 1 },
+			],
+			domain: [0, 5],
+			dataKey: 'axis',
+		};
+
+		// when
+		const wrapper = mount(
+			<RadarChart data={props.data} domain={props.domain}>
+				<PolarAngleAxis dataKey={props.dataKey} />
+				<Radar
+					name="Trust score"
+					dataKey="A"
+					dot
+					stroke="#19426c"
+					fill="#19426c"
+					fillOpacity={0.1}
+				/>
+			</RadarChart>,
+		);
+
+		expect(wrapper.getElement()).toMatchSnapshot();
+	});
+	it('should render a chart with clickable labels', () => {
+		const props = {
+			data: [
+				{ axis: 'Validity', A: 4 },
+				{ axis: 'Social curation', A: 3 },
+				{ axis: 'Completeness', A: 2 },
+				{ axis: 'Discoverability', A: 1 },
+				{ axis: 'other', A: 1 },
+			],
+			domain: [0, 5],
+			dataKey: 'axis',
+			activeAxis: 2,
+			func: true,
+		};
+
+		// when
+		const wrapper = mount(
+			<RadarChart data={props.data} domain={props.domain}>
+				<PolarAngleAxis
+					dataKey="axis"
+					tick={<LabelWithClick activeAxis={props.activeAxis} />}
+					onClick={props.func}
+				/>
+				<Radar
+					name="Trust score"
+					dataKey="A"
+					dot={false}
+					stroke="#19426c"
+					fill="#19426c"
+					fillOpacity={0.1}
+				/>
+			</RadarChart>,
+		);
+
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
In this test you should not use snapshots.
In few words, use only snapshots to validate the overall markup. Otherwise, select the specific elements and assert them. The reason is that snapshoting everything everytime makes it hard to see the snapshot diff and what has changed compared to what the test is supposed to assert. That makes the snapshot hard to maintain and can trigger false negative on specific tests if the markup has changed.

Here you can:
- mount the chart
- select a clickable element
- simulate a click `element.simulate('click')` and check that the expected callback is called